### DISCUSSION TITLE
Install Plasmoid

### DISCUSSION THREAD

**Discussion by J-Cake** [Martchus/syncthingtray#297] [Q&A]
Hey sorry if this is super basic, I'm having trouble understanding the docs.

I'd like to install the Plasmoid under Arch for Plasma 6.

I tried installing using `yay -Sy syncthingtray`, but am failing to find any of the Dolphin or Plasmoid files I need. 

According to [Further Dependencies](https://github.com/Martchus/syncthingtray?tab=readme-ov-file#further-dependencies), I need to add CMake variables. How do I do this?

Thanks

**Comment by Martchus** [comment]
You need to install `syncthingtray-qt6`.
